<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerARnavExtensionModule</class>
 <widget class="qSlicerWidget" name="qSlicerARnavExtensionModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="PickPoint">
          <property name="text">
           <string>PickPoint</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboPointTool">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SurfacePick">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>SurfacePick</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SurfaceRegistration">
          <property name="text">
           <string>SurfaceRegistration</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="PointShow">
          <property name="text">
           <string>PointShow</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PrePointDelete">
          <property name="text">
           <string>PrePointDelete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AllPointsDelete">
          <property name="text">
           <string>AllPointsDelete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="START_TRACKING">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>START_TRACKING</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="STOP_TRACKING">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>STOP_TRACKING</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CALIBRATE">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>CALIBRATE</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OBTAIN_ANGLE">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>OBTAIN_ANGLE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="START_OPENIGT">
          <property name="text">
           <string>START_OPENIGT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="PROBE_STATUS">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>PROBE_STATUS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="SURG_TOOL_STATUS">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>SURG_TOOL_STATUS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="CALIBRATION_STATUS">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>CALIBRATION_STATUS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="REF_STATUS">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>REF_STATUS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CreateProbeXml">
          <property name="text">
           <string>CreateProbeXml</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CreateToolXml">
          <property name="text">
           <string>CreateToolXml</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Calibrated:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCaliTool">
          <item>
           <property name="text">
            <string>Probe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Surgical Tool</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="Label_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
